core/src/trezor/lvglui/i18n/locales/zh_hk.py

```
@@ -24,7 +23,7 @@
     "警告",
     "跳過本次備份？可以另選時間備份您的 OneKey，但只能備份一次。",
     "請將助記詞保存在安全的地方，切勿將其發送給任何人。",
-    "接下來將展示一組單詞，稱為助記詞。在此之前，您需要了解：",
+    "下一個屏幕將顯示您的恢複短語。以下是您需要了解的內容：",
```

Ruff (0.14.3)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)